### DISCUSSION TITLE
How to transfer discussion into an issue in Neurodesktop, Neurocommand or Neurocontainers?

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#126] [Q&A] [answered]
I tried to press on "Transfer Discussion" on the right, typed neurodesktop (I didn't get any options in the drop-down menu), and got an error screen.
Do I need permission for that? If yes, can you give me one?

There is a discussion that I want to assign to myself, and to my knowledge, only issues can be assigned.

Best,
Oren

**Comment by stebo85** [comment, marked as answer]
Dear @civier 

you first need to convert a disucssion to an issue then move the issue. Or even easier: create an issue in the right repo from the start.

Kind Regards
Steffen

**Reply by aswinnarayanan** [reply]
Transfer discussion is for moving a discussion from neurodesk.github.io/discussions to another repository's discussion page (The NeuroDesk Org doesnt have any other ones, so you're not able to find any).
You'll need to create an issue from a discussion like stebo85 mentioned
